server/templates/mainPages/index.html

```diff
@@ -3,25 +3,91 @@
 {% block navbar %}{% endblock%}
 {% block content %}
 <div class="container">
-<h1>Home</h1>
-<p>Home will have general information about us and the project. This includes Background Information, an Overview, and Instructions</p>
-<br>
-<p>Background Information
-<br>
-<p>Toolkit Overview
-<br>
-<p>Instructions
-<br>
-<p>About Us</p>
-<div class="row">
-  <div class="column">
-    <div id="center"><p>Valeda and Eric...</p></div>
+  <center>
```

Comment:
<center> tag was depreciated in html5 and a better way to center divs would be to give it an <div id="center"></div> and include css that centers the element that way.